rmw_implementation/src/functions.cpp

```diff
@@ -81,16 +84,19 @@ get_library()
 void *
 get_symbol(const char * symbol_name)
 {
-  Poco::SharedLibrary * lib = get_library();
+  rcutils_shared_library_t * lib = get_library();
```

Comment:
This seems weird to me that it is making a new shared library handle each time. I guess it's essentially a noop for dlopen and the related functions, but I wonder if poco was making this into a single for us... Perhaps this should be shard resource rather than being loaded each time `get_symbol` is called?

Also, when does this shared library get unloaded? never?

Reply:
Library is declared here as `static` 
https://github.com/ros2/rmw_implementation/blob/da6ca0af16b57469fa88df0566eababe8b8fb947/rmw_implementation/src/functions.cpp#L52-L55

Then it's checking each time if the library is already load.

This code loads `rmw_X_cpp.so` library I think this library should be released when :
 - the node is killed?
 - ?